### PR TITLE
Add "create tag" in select input

### PR DESCRIPTION
**Pull request by SimonLab**
Please feel free to discard/close this PR, see https://github.com/dwyl/mvp/issues/221#issuecomment-1332004568

It is in example how a tag can be created from the select input.
As mentioned in the issue, this is not a prefect UX, this PR can be used for testing and discussion.

### CODE REVIEW

File: lib/app_web/live/app_live.html.heex [hunk 1]
```diff
@@ -85,6 +85,11 @@
               </div>
             </li>
           <% end %>
+          <%= if Enum.empty?(@tags) and String.length(String.trim(@search_tag)) != 0 do %>
```
Comment (SimonLab) [inline lib/app_web/live/app_live.html.heex:88]: Only show the "create" action when the search for a tag returns 0 results

**Comment by nelsonic** [comment]
@SimonLab feels like we need "review apps" to discuss ... https://github.com/dwyl/learn-devops/issues/85 💭

**Comment by SimonLab** [comment]
Yes I will look at how we can create review apps with Fly.
In the meantime I've created the PR to allow people to test it on localhost

**Comment by SimonLab** [comment]
image

Clicking on the "create my new tag" will create the tag and select it before resetting the search tag input:
image

**Comment by nelsonic** [comment]
The concept of having to _manually_ "Create new tag" is a UX antipattern that takes the person out of the flow of "Capture" mode and into "admin". We aren't going to do this as outlined in: https://github.com/dwyl/mvp/issues/221#issuecomment-1404384279

**Comment by SimonLab** [comment]
To clarify if we need to search code from this PR, the feature added here doesn't redirect to another page. The create link call some liveview code which creates the tag and update the UI with the tag pill

**Pull request closed without merging**